tests/sles4sap/trento/test_hana_restore_stopped.pm

```diff
@@ -32,7 +33,8 @@ sub run {
 
     # Restart the stopped instance
     my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
-    qesap_ansible_cmd(cmd => "sudo crm resource refresh rsc_SAPHana_HDB_HDB00 $primary_host",
+    # vmhana01 hardcoded in place of $primary_host as the second one is only valid for Ansible
+    qesap_ansible_cmd(cmd => "sudo crm resource refresh rsc_SAPHana_HDB_HDB00 vmhana01",
```

Comment:
No it will not, and the comment is implicitly about it.

Reply:
Yes, I get you're using `vmhana01` instead of `$primary_host` as that variable contains `"hana[0]"` which is only visible to ansible and which is addressed in the comment, but `vmhana01` itself will not work either in GCP, right? Only in Azure and EC2. If that's the case, can't we get the actual hostname of node 1 from the inventory and use it for GCP (or all 3)?

Reply:
I created https://jira.suse.com/browse/TRNT-2029 about it